src/adapter/src/catalog.rs

```diff
@@ -2445,6 +2461,25 @@ impl<S: Append> Catalog<S> {
         for (name, value) in system_config {
             catalog.state.insert_system_configuration(&name, &value)?;
         }
+        if !catalog.state.system_config().config_has_synced_once() {
+            if let Some(system_parameter_frontend) = config.system_parameter_frontend {
+                let system_config = catalog.state.system_config();
+                let mut params = SynchronizedParameters::new(system_config.clone());
+
+                if system_parameter_frontend.start_and_initialize().await {
```

Comment:
Wait, this doesn't seem right? `start_and_initialize` looks like it can fail? But don't we want to use `ensure_initialized` here instead, to actually wait for successful initialization?

Reply:
> Wait, this doesn't seem right?

I think I wanted to ensure that we don't block new environments indefinitely if for some reason LD is down. I'll use `ensure_initialized` and we can see if this is a problem in practice.